app/handlers/add_unit_to_new_ticket_handler.rb

```diff
@@ -0,0 +1,32 @@
+class AddUnitToNewTicketHandler < BaseHandler
+  def execute!
+    @unit = Unit.find_by(id: message.data)
+
+    Ticket.create(name: 'Telegram Bot Ticket',
+                  description: 'Some Ticket Bot Description',
+                  user: telegram_profile.user,
+                  unit: @unit,
+                  status: 2)
```

Comment:
Використовуй ключові слова в енумах.
```suggestion
                  status: :draft)
```